cmd/kagome/server/cmd.go

```diff
@@ -191,14 +196,97 @@ type TokenizeDemoHandler struct {
 	tokenizer tokenizer.Tokenizer
 }
 
-func (h *TokenizeDemoHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
-	type record struct {
-		Surface        string
-		Pos            string
-		Baseform       string
-		Reading        string
-		Pronounciation string
+type record struct {
+	Surface        string
+	Pos            string
+	Baseform       string
+	Reading        string
+	Pronounciation string
+}
+
+func (h *TokenizeDemoHandler) analyze(sen, mode string) (string, []record, error) {
```

Comment:
__[gometalinter]__
* [gocyclo] cyclomatic complexity 16 of function (*TokenizeDemoHandler).analyze() is high (> 10)